bodhi/server/util.py

```diff
@@ -684,3 +686,101 @@ def sort_severity(value):
     }
 
     return value_map.get(value, 99)
+
+
+def get_critpath_pkgs_from_pdc(branch):
+    """
+    Searches PDC for critical path packages based on the specified branch
+    :param branch: string representing the branch
+    :return: a list of critical path packages
+    """
+    pdc_api_url = '{}/rest_api/v1/component-branches/'.format(
+        config.get('pdc_url', 'https://pdc.fedoraproject.org/').rstrip('/'))
```

Comment:
With the new config PR I made we can move this default to ```bodhi.server.config.BodhiConfig._defaults```, once #1544 is merged.